testsuite/features/step_definitions/command_steps.rb

```diff
@@ -21,11 +21,12 @@
 
 Then(/^reverse resolution should work for "([^"]*)"$/) do |host|
   node = get_target(host)
-  initial_time = Time.now
-  result, return_code = node.run("getent hosts #{node.full_hostname}", check_errors: false)
-  end_time = Time.now
-  resolution_time = end_time - initial_time
-  result.delete!("\n")
+  result, return_code = node.run("date +%s; getent hosts #{node.full_hostname}; date +%s", check_errors: false)
+  lines = result.split("\n")
+  initial_time = lines[0]
+  result = lines[1]
+  end_time = lines[2]
+  resolution_time = end_time.to_i - initial_time.to_i
```

Comment:
`date +%s` returns an integer, not a float.

It can impact only to be more tolerant if I am not wrong. For example, we start at 15.1 and end at 17.9. It's 2.8 seconds, but will be computed as 17 - 15 = 2 seconds.